incubator/helm-release-pruner/templates/cronjob.yml

```diff
@@ -33,6 +33,9 @@ spec:
             - mountPath: /usr/local/bin/prune-releases.sh
               name: config
               subPath: prune-releases.sh
+            env:
+            - name: TILLER_NAMESPACE
+              value: {{ .Values.tiller.namespace }}
```

Comment:
Can you add a default value for the namespace in the values file?

Reply:
Sure.  I added kube-system as a default.